src/model/users.rs

```diff
@@ -258,24 +277,33 @@ pub struct CurrentAccountData {
 	#[serde(default)]
 	/// Can presumably be an empty string
 	pub obfuscated_email: String,
-	#[serde(default)]
-	/// Can be an empty string
-	pub obfuscated_pending_email: String,
 	/// Can be an empty string
 	#[serde(default)]
-	pub oculus_id: String,
-	#[serde(default)]
+	pub obfuscated_pending_email: String,
 	/// Can be empty
-	pub past_display_names: Vec<Either<PastDisplayName, String>>,
```

Comment:
I assume at one point it was a list of strings instead of a list of objects, but for some reason using Either here causes it to fail to serialize, I don't know why.